### DISCUSSION TITLE
Turn off all businesses? I only want to see streets

### DISCUSSION THREAD

**Discussion by ackern** [organicmaps/organicmaps#1676] [Q&A]
How do I stop the map from showing businesses (all of them)? I need a street map, just a regular street map, not a business directory.

**Comment by biodranik** [comment]
We are discussing now an implementation of different styles for different purposes, with different POI visibility. What is your use-case? Are you navigating by car or by foot in the city?

**Reply by ackern** [reply]
> Are you navigating by car or by foot in the city?

I'm not navigating at all, if you mean "navigating" as the word has come to be used, which is directions provided in real time that follow your location and tell you when and where to turn. I just want a map that I can look at and see the streets, like a map on paper. For outlines and labels of buildings/business/organizations, if I had to choose between none at all and distracting ones I'd much rather have none at all because streets are the main thing for me. openstreetmap.org is pretty good in all these respects. When you zoom out you go from seeing all the businesses/institutions labeled (but the size and color are unobtrusive), to just the larger ones labeled and a few have symbols (cross for church etc.), to none have labels and fewer have symbols, to none have symbols but the streets are still all labeled, and so on as you zoom out further. Or at least this is what I see on my Windows laptop computer.  I guess ideally it'd be nice to be able to turn labels on/off at will but I don't think openstreetmap.org has that.

**Reply by biodranik** [reply]
Thanks for the details. We definitely need to tune our current map styles and the engine. Please share your suggestions in https://github.com/organicmaps/organicmaps/issues/1577
